docker-plugin/src/main/java/com/nirima/jenkins/plugins/docker/DockerTemplateBase.java

```diff
@@ -1,5 +1,6 @@
 package com.nirima.jenkins.plugins.docker;
 
+import com.github.dockerjava.api.model.*;
```

Comment:
> [Minor] Think rule to not use wildcard imports is good. (In idea it can be found on `code style` -> `java` -> `imports`)

When review, you can see each class in scope and where from every function or const is imported. Also any import modification is visible on review